source/deployment-paradigms/multi-region.txt

```diff
@@ -102,33 +229,15 @@ and ``eu-west-3``). This ensures data sovereignty since all regions are within
 the EU, while offering high availability if there's a regional outage that 
 affects one of the nodes.
 
-.. _arch-center-multi-region-considerations:
-
-Considerations for Multi-Region Deployments
--------------------------------------------
-
-Other considerations for multi-region deployments include:
-
-- High availability depends on the deployment of nodes across
-  regions as well as the number, distribution, and priority order of
-  nodes. To learn more about recommended
-  {+cluster+} topologies for high availability, see
-  :ref:`arch-center-high-availability`.
-- Multi-region deployments are available only for ``M10`` dedicated
-  {+clusters+} and larger.
-- High availability and low latency also depend on where you deploy 
-  your application. When optimizing for availability and latency, plan the 
-  deployment of your entire application stack, not just the database.
-
-For more considerations, see 
-:atlas:`Considerations 
-</cluster-config/multi-cloud-distribution/#considerations>` in the
-{+service+} documentation.
-
 .. _arch-center-multi-region-rec-summary:
 
 Recommendations for Multi-Region Deployments
 ---------------------------------------------
 
 .. include:: /includes/rec-list.rst
 
+
+.. toctree::
+   :titlesonly:
+
+   Multi-Region Latency Strategies </deployment-paradigms/latency-strategies>
```

Comment:
Suggestion: My preference for this TOC label would be:

```suggestion
   Latency Reduction Strategies </deployment-paradigms/latency-strategies>
```
Since they already have the context of multi-region from the parent page, and this gets at the crux of latency reduction. The full title can stay on the page!